[2025-04-16T18:53:47.802+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-16T18:53:47.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: plusvalia_pipeline.extract_avaluos manual__2025-04-16T18:48:41.144194+00:00 [queued]>
[2025-04-16T18:53:47.872+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: plusvalia_pipeline.extract_avaluos manual__2025-04-16T18:48:41.144194+00:00 [queued]>
[2025-04-16T18:53:47.873+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-04-16T18:53:47.914+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_avaluos> on 2025-04-16 18:48:41.144194+00:00
[2025-04-16T18:53:47.931+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'plusvalia_pipeline', 'extract_avaluos', 'manual__2025-04-16T18:48:41.144194+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/plusvalia_pipeline_dag.py', '--cfg-path', '/tmp/tmpc13xi_je']
[2025-04-16T18:53:47.941+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask extract_avaluos
[2025-04-16T18:53:47.940+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=235) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-16T18:53:47.943+0000] {standard_task_runner.py:63} INFO - Started process 237 to run task
[2025-04-16T18:53:48.025+0000] {task_command.py:426} INFO - Running <TaskInstance: plusvalia_pipeline.extract_avaluos manual__2025-04-16T18:48:41.144194+00:00 [running]> on host 7dfd617159b9
[2025-04-16T18:53:48.232+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Brayan Miranda' AIRFLOW_CTX_DAG_ID='plusvalia_pipeline' AIRFLOW_CTX_TASK_ID='extract_avaluos' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T18:48:41.144194+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T18:48:41.144194+00:00'
[2025-04-16T18:53:48.234+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-16T18:53:48.254+0000] {logging_mixin.py:188} INFO - Extrayendo aval√∫os desde PDF...
[2025-04-16T18:53:48.258+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-16T18:53:48.259+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/etl/extract_avaluos.py", line 25, in extract_avaluos
    pdf_files = list(pdf_dir.glob("*.pdf"))
                     ^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'glob'
[2025-04-16T18:53:48.276+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=plusvalia_pipeline, task_id=extract_avaluos, run_id=manual__2025-04-16T18:48:41.144194+00:00, execution_date=20250416T184841, start_date=20250416T185347, end_date=20250416T185348
[2025-04-16T18:53:48.308+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task extract_avaluos ('str' object has no attribute 'glob'; 237)
[2025-04-16T18:53:48.333+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-04-16T18:53:48.368+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-16T18:53:48.388+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
